web/src/components/ApdList.js

```diff
@@ -68,6 +60,14 @@ const ApdList = (
         <main id="start-main-content">
           <div className="ds-u-padding-top--2">
             <UpgradeBrowser />
+            {error && (
+              <div className="ds-c-alert ds-c-alert--error">
```

Comment:
You could use the react component version of this `<Alert>` from design.cms.gov here. 
https://design.cms.gov/components/alert/